docs/api-reference/core/layer.md

```diff
@@ -238,13 +238,16 @@ Specifies a longitude and a latitude from which meter offsets are calculated. Se
 
 ##### `wrapLongitude` (Boolean, optional)
 
-Automatically wraps longitudes over the 180th antimeridian.
+- Default: `false`
+
+Normalizes geometry longitudes. Only works with geospatial data.
```

Comment:
> Only works with geospatial data.

Nit: This is perhaps a bit vague... Should this list certain coordinate systems or that is too complex?